locksmith/src/routes/index.ts

```diff
@@ -22,8 +22,7 @@ router.use((request, _, next) => {
     // When the route starts with the chain (deprecated?)
     chain = parseInt(match[1])
   } else if (request.query?.chain) {
-    // @ts-expect-error
-    chain = parseInt(request.query.chain)
+    chain = parseInt(String(request.query.chain))
```

Comment:
@searchableguy there is a possibility that this gives `NaN` if `request.query.chain` has a value thas is not a number 
```suggestion
    const value = String(request.query.chain)
    chain = !isNaN(value) ? parseInt(value) : undefined
```

what do you think?

Reply:
Looks okay.